cli/pcluster/createami.py

```diff
@@ -308,8 +308,6 @@ def create_ami(args):
 
     # Ensure AWS_DEFAULT_REGION env is set.
     # Set it explicitly if the CLI arg was passed. Otherwise let the config object do it.
-    if args.region:
-        os.environ["AWS_DEFAULT_REGION"] = args.region
```

Comment:
I guess this code never worked, but createami command seems to accept a -r/--region option.
Should we remove that option from createami, or somehow make it work?

Reply:
i thought the same thing, bu this code was. redundant. there's a conditional in cli.py::main that does this for any subcommand that supports a region arg.